src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -211,6 +212,129 @@ function getUnitTranslationKey(unit: Unit): TranslationPaths {
     return unitTranslationKeysStrategy[unit];
 }
 
+/**
+ * @param error workspace change owner error
+ * @returns request ownership change parameters object
+ */
+function getOwnershipChecks(error: ValueOf<typeof CONST.POLICY.OWNERSHIP_ERRORS>) {
```

Comment:
I don't think we need this method. I think we should instead retrieve it directly from the policy in actions/Policy.ts. I left a comment about it below

Reply:
addressed